website/content/docs/api-gateway/usage/basic-usage.mdx

```diff
@@ -0,0 +1,131 @@
+| `-k8-context`          | String value that specifies the Kubernetes context to use when starting the Consul server.                                                                                            | Optional | current context                                                         |
+| `-k8-namespace`        | String value that specifies the Kubernetes namespace to use when starting the Consul server.                                                                                          | Optional | `default`                                                               |
+| `-log-json`            | Boolean value that enables or disables JSON format for the log output.                                                                                                                | Required | `false`                                                                 |
+| `-log-level`           | String value that specifies the logging level. The following values are supported: <br/>- `trace` (highest level of detail) <br/>- `debug` <br/>- `info` <br/>- `warn` <br/>- `error` | Optional | `info`                                                                  |
+| `-metrics-port`        | Integer value that specifies the port number for collecting metrics.                                                                                                                  | Optional | none                                                                    |
+| `-pprof`               | Integer value that specifies the Go pprof port number for collecting metrics.                                                                                                         | Optional | none                                                                    |
+| `-sds-server-host`     | String value that specifies the host server for the secret discovery service (SDS).                                                                                                   | Optional | `consul-api-gateway-controller.default.`<br/>`svc.cluster.`<br/>`local` |
+| `-sds-server-host`     | Integer value that specifies the port number for the secret discovery service (SDS).                                                                                                  | Optional | `9090`                                                                  |
+
+You can also issue the `version` command to print the Consul API Gateway version to the console:
+
+```shell-session
+$ ./consul-api-gateway version
+consul-api-gateway 0.1.0
+```
+--->
+
+## Common Error Messages
+
+Some of the errors messages commonly encountered during installation and operations of Consul API Gateway are listed below, along with suggested methods for resolving them.
+
```

Comment:
```suggestion
## Error Messages

```
A few things - 

First, I think we should just talk about "error messages" as opposed to "common error messages" because I don't want to create the impression that people are always running into a lot of errors. It's a bit rhetorical, but if the idea is to create a catalog of error messages, describing them as _common_, IMO, implies that errors are common.  

Second, I suggest trimming the first paragraph because it doesn't seem to add a lot of value. 

Third, I think I was wrong about suggesting that this section be merged into this page. I think it's fine for now, but when we build the catalog of error messages it might make sense for them to exist as a standalone topic. I'm open to discussion.